tests/Functional/Bug/Bug256Test.php

```diff
@@ -29,7 +27,7 @@ class Bug256Test extends AbstractConnectionTest
 
     protected $channel2;
 
-    public function setUp()
+    protected function setUpCompat()
```

Comment:
I would suggest to use more meaningful method name, let's `initialize`. `setUpCompat` sounds like suspicious workaround :)

Reply:
I strongly disagree.
1. `setUp` and `rearDown` methods - are standard phpunit's methods => any php developer who did tests at least once knows them. `initialize` - is something not standard. Every new developer who would want to contribute to this repo would have to learn "local" namings.
2. `Compat` pattern is pretty common. For instance, google uses it for [android compatibility libraries](https://developer.android.com/topic/libraries/support-library/packages). Adding `Compat` word to the end of the method/class names is self-explaining that it has something to do with compatibility.

Instead I:
- added phpdocs to these two methods so developers would see explanation why these methods exist.
- renamed `TestCase` to `TestCaseCompat` to avoid confusion between default phpunit's `TestCase` and "compat" `TestCase`.

Reply:
even more "compats", nice :D this phpunit comaptibility layer is temporal, I will drop it later on.